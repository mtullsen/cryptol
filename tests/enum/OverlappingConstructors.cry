enum Foo = A | B

f : Foo -> ()
f x =
  case x of
    A -> ()
    B -> ()
    A -> ()
