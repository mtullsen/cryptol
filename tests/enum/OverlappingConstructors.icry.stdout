Loading module Cryptol
Loading module Cryptol
Loading module Main

[error] at OverlappingConstructors.cry:5:3--10:0:
  Overlapping choices for constructor A:
    Pattern at OverlappingConstructors.cry:8:5--8:6
    Pattern at OverlappingConstructors.cry:6:5--6:6
